code/modules/mob/living/carbon/xenomorph/Xenomorph.dm

```diff
@@ -567,6 +567,12 @@
 		if(LIGHTING_PLANE_ALPHA_VISIBLE)
 			return XENO_VISION_LEVEL_NO_NVG
 
+/mob/living/carbon/Xenomorph/proc/get_life_kills()
+	return life_kills_total
+
+/mob/living/carbon/Xenomorph/proc/get_life_damage_taken()
+	return life_damage_taken_total
```

Comment:
don't use procs to read variables unless it's extremely likely to need an override in the future. procs have something called proc overhead so it's better to just read variables directly

Reply:
donezo